### PR TITLE
Debug panic of sincelastcommit

### PR DESCRIPTION
**Pull request by kachick**
~I think this is the panic cause of #134~

### CODE REVIEW

File: .github/workflows/notify_long_time_no_see.yaml [hunk 1]
```diff
@@ -18,9 +18,9 @@ jobs:
     - uses: actions/checkout@v3
     - name: install sincelastcommit
       run: |
-        cd ./tool/sincelastcommit
+        pushd ./tool/sincelastcommit
         go install .
-        cd ..
```
Comment (kachick) [inline .github/workflows/notify_long_time_no_see.yaml:23]: When keeping cd, this should be `cd ../../`

**Comment by kachick** [comment]
😮 https://ebc-2in2crc.hatenablog.jp/entry/2023/01/13/181801

**Review by pankona — APPROVED** [review]
is this necessary?

**Comment by kachick** [comment]
Will use after tasks!

**Pull request merged**